glide.yaml

```diff
@@ -0,0 +1,18 @@
+package: github.com/turbonomic/mesosturbo
+import:
+- package: github.com/golang/glog
+- package: github.com/spf13/pflag
+- package: github.com/turbonomic/turbo-go-sdk
```

Comment:
I think we can specify the turbo-go-sdk version here, so that we know clearly the version of the turbo-go-sdk.

For example:
```yaml
- package: github.com/turbonomic/turbo-go-sdk
  version: 03603dc59c3a7d0e3552f9f98b590000b5e5c987
```